galite-core/src/main/kotlin/org/kopi/galite/util/PrintJob.kt

```diff
@@ -68,33 +72,32 @@ class PrintJob(var dataFile: File, var delete: Boolean, var format: Rectangle) {
   }
 
   /**
-   * OutputStream has to be closed before using inputStream
+   * outputStream has to be closed before using inputStream
    * use with care, know what you do!
    */
-  fun getOutputStream(): OutputStream = FileOutputStream(dataFile)
+  val outputStream: OutputStream get() = FileOutputStream(dataFile)
 
   /**
-   * outputStream has to be closed before using inputStream
+   * outputStream has to be closed before using getInputStream
    */
-  fun getInputStream(): InputStream = FileInputStream(dataFile)
+  val inputStream: InputStream get() = FileInputStream(dataFile)
```

Comment:
Why did you change functions to vals with custom getters when we don't have properties with "outputstream" and "inputstream" names in Printjob class ?